src/components/widgets/sales-cycle/sales-cycle.directive.coffee

```diff
@@ -22,25 +22,22 @@ module.controller('WidgetSalesCycleCtrl', ($scope, $q, ChartFormatterSvc, $filte
   # Widget specific methods
   # --------------------------------------
   w.initContext = ->
-    if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.status_average_durations)
+    dhb = ImpacDashboardsSvc.getCurrentDashboard()
+    status_selection = w.metadata.status_selection || dhb.metadata.status_selection || { values: [] }
```

Comment:
we use the dashboard metadata as a source of information if no status_selection is set for the widget, whatever the reach is (for a new widget for example)